CourseGrab/ViewControllers/Home/HomeTableViewController.swift

```diff
@@ -339,7 +343,7 @@ extension HomeTableViewController {
             case .value(let response):
                 guard response.success else { return }
                 let change = self.removeSectionFromModel(response.data)
-                let description = "\(section.subjectCode) \(section.courseNum): \(section.title) - \(section.section)"
```

Comment:
Since we want Analytics to be uniform regardless of timezone probably best to keep `description` equal to`section.section` for all of these logging events.